src/ngAnimate/animateQueue.js

```diff
@@ -454,9 +454,9 @@ var $$AnimateQueueProvider = ['$animateProvider', function($animateProvider) {
     }
 
     function clearElementAnimationState(element) {
-      element = element.length ? element[0] : element;
-      element.removeAttribute(NG_ANIMATE_ATTR_NAME);
-      activeAnimationsLookup.remove(element);
+      var node = getFirstNode(element);
+      node.removeAttribute(NG_ANIMATE_ATTR_NAME);
+      activeAnimationsLookup.remove(node);
     }
 
     function isMatchingElement(a,b) {
```

Comment:
What are `a` and `b` here—jQuery instances of DOM nodes?  Do they need to be wrapped/unwrapped before calling isMatchingElement at all call sites?